setup.sh

```diff
@@ -28,6 +28,10 @@ GRANT ALL PRIVILEGES ON \`patchdemo\_%\`.* TO 'patchdemo'@'localhost';
 composer update --no-dev
 npm install --production
 
+# set session expiration to a month (default is 24 minutes???), cookie expiration too
```

Comment:
This sounds like 24*60 instead of 24*60*60: in good PHP manners someone forgot to convert minutes to seconds.
And indeed it is: https://stackoverflow.com/questions/14703363/why-is-the-standard-session-lifetime-24-minutes-1440-seconds/37176824#37176824